lib/AbstractHandle/Utils/MongoUtil.py

```diff
@@ -5,26 +5,12 @@
 import subprocess
 import traceback
 
+MONGO_COLLECTION = 'handle'
+MONGO_HID_COUNTER_COLLECTION = 'handle_id_counter'
```

Comment:
I don't think there's any reason for these to be configurable since the handle service owns the database

Reply:
👍